modules/react/button/lib/DropdownButton.tsx

```diff
@@ -1,68 +1,51 @@
 import * as React from 'react';
 import {caretDownIcon} from '@workday/canvas-system-icons-web';
-import {GrowthBehavior, useTheme, Themeable} from '@workday/canvas-kit-react/common';
-import {DropdownButtonVariant, ButtonIconPosition, ButtonOrAnchorComponent} from './types';
+import {
+  GrowthBehavior,
+  useTheme,
+  Themeable,
+  createComponent,
+} from '@workday/canvas-kit-react/common';
+import './types';
```

Comment:
Is this necessary?

Reply:
heh. Oops. All invalid imports were removed, but nothing checks an empty import. This could happen if only Variants are imported, but this is very unlikely in application code. It won't hurt if this happens, but I'll remove that here.